### PR TITLE
new icon: chakraui (original, plain, original-wordmark, plain-wordmark)

### LINKED ISSUE #578: Icon request: Chakra UI
### Links

Chakra UI: https://chakra-ui.com/

### PR DESCRIPTION
**Pull request by ruubytes**
## Double check these details before you open a PR

- [X] PR does not match another non-stale PR currently opened
- [X] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [X] PR's base is the `develop` branch.
- [X] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [X] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [X] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #578 #1597**

## Link to prove your SVG is correct and up-to-date.

https://chakra-ui.com/

### Additional information
The original has a gradient so it doesn't necessarily fit the laid out SVG standards, the circle background references the linear gradient id in its fill attribute. I've kept it similar to the bootstrap-original.svg.

The plain version color is the mean from the gradient.

### CODE REVIEW

File: icons/chakraui/chakraui-plain.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128">
+  <path
+    d="M64,0C28.65,0,0,28.65,0,64s28.65,64,64,64,64-28.65,64-64S99.35,0,64,0Zm29.56,57.9l-.02,.02s0,0-.01,.01l-49.13,46.38h0c-.2,.2-.48,.32-.79,.32-.62,0-1.13-.5-1.13-1.13,0-.26,.09-.49,.23-.68,0,0,22.69-31.38,23.15-32,.46-.63,1.13-1.71-.75-1.71h-29.97c-.53,0-.96-.43-.96-.96,0-.27,.12-.52,.3-.69h0S78.42,23.69,78.42,23.69h0s0,0,0,0h0c.2-.19,.46-.31,.76-.31,.6,0,1.08,.49,1.08,1.08,0,.17-.04,.32-.11,.46h0s-15.5,28.27-16.11,29.4c-.61,1.13-.89,2.04,.82,2.04h28.02c.51,0,.92,.41,.92,.92,0,.24-.1,.46-.25,.62h0Z"
+    fill="#52c8c5" />
```
Comment (Snailedlt) [inline icons/chakraui/chakraui-plain.svg:4]: I think we can remove the fill color here
@lunatic-fox Is fill preferred or not for plain and line versions?
```suggestion
    d="M64,0C28.65,0,0,28.65,0,64s28.65,64,64,64,64-28.65,64-64S99.35,0,64,0Zm29.56,57.9l-.02,.02s0,0-.01,.01l-49.13,46.38h0c-.2,.2-.48,.32-.79,.32-.62,0-1.13-.5-1.13-1.13,0-.26,.09-.49,.23-.68,0,0,22.69-31.38,23.15-32,.46-.63,1.13-1.71-.75-1.71h-29.97c-.53,0-.96-.43-.96-.96,0-.27,.12-.52,.3-.69h0S78.42,23.69,78.42,23.69h0s0,0,0,0h0c.2-.19,.46-.31,.76-.31,.6,0,1.08,.49,1.08,1.08,0,.17-.04,.32-.11,.46h0s-15.5,28.27-16.11,29.4c-.61,1.13-.89,2.04,.82,2.04h28.02c.51,0,.92,.41,.92,.92,0,.24-.1,.46-.25,.62h0Z" />
```

**Review by Snailedlt — DISMISSED** [review]

**Review by lunatic-fox — APPROVED** [review]
Great work! ✔
Thank you for this contribution! 🚀

**Review by Panquesito7 — DISMISSED** [review]
You might want to add `wordmark` versions as well (I found one on the [GitHub repository](https://github.com/chakra-ui/chakra-ui) of the project).
The other icons are looking great, thanks! 🚀

**Review by lunatic-fox — APPROVED** [review]
Everything seems to be ok right now. 👍🏼
Thank you for this contribution! 🚀
image
image

**Review by canaleal — APPROVED** [review]
Excellent 👍 
All is good.

**Pull request merged**